src/components/dashboard/users/LnkEditUserPreferences.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from 'react';
+import globalize from '../../../scripts/globalize';
+
+type IProps = {
+    title?: string;
+    className?: string;
+}
+
+const createLinkElement = ({ className, title }) => ({
+    __html: `<a
+        is="emby-linkbutton"
+        class="${className}"
+        href='#'
+        >
+        ${title}
+    </a>`
+});
+
+const LnkEditUserPreferences: FunctionComponent<IProps> = ({ className, title }: IProps) => {
```

Comment:
It looks like this should be "Link". The file name has the same typo.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3070/commits/1a66afa6104ce9c68bac300626a35f5d1426a15f